browser/ui/webui/brave_new_tab_message_handler.h

```diff
@@ -1,4 +1,4 @@
-// Copyright (c) 2019 The Brave Authors. All rights reserved.
+// Copyright (c) 2020 The Brave Authors. All rights reserved.
```

Comment:
same as above, why are we changing these?

Reply:
reverted